querydsl-apt/src/main/java/com/querydsl/apt/DefaultConfiguration.java

```diff
@@ -177,6 +181,19 @@ public DefaultConfiguration(
             }
         }
 
+        if (options.containsKey(QUERYDSL_VARIABLE_NAME_FUNCTION_CLASS)) {
+            try {
+                variableNameFunction = (Function<EntityType, String>) Class.forName(options.get(QUERYDSL_VARIABLE_NAME_FUNCTION_CLASS)).newInstance();
+            } catch (RuntimeException e) {
+                throw e;
+            } catch (Exception e) {
+               variableNameFunction = new DefaultVariableNameFunction();
```

Comment:
@timowest What is your take on this? I think a syserr statement isn't that bad (it's only on exceptional behavior), or should we add a logger to that class?